<launch>

  <node name="stripmap_planner_2d"
    pkg="mav_coverage_planning_ros"
    type="stripmap_planner_2d_exact_node"
    output="screen"
    clear_params="true" >

    <!-- Basic and control parameter -->
    <rosparam command="load" file="$(find mav_coverage_planning_ros)/cfg/base_planner_2d.yaml" />
    <!-- Polygon, altitude, and world frame ID -->
    <rosparam command="load" file="$(find mav_coverage_planning_ros)/cfg/example_polygon.yaml" />
    <!-- Sweep planner specific parameter -->
    <param name="lateral_fov" value="1.57" />
    <param name="longitudinal_fov" value="1.0472" />
    <param name="min_view_overlap" value="0.2" />
  </node>

</launch>
